### PR TITLE
libutils: isoc: implement more ctypes functions

### PR DESCRIPTION
**Pull request by blukat29**
Implement missing functions from ctypes.h.

Signed-off-by: Yunjong Jeong <yunjong@kaist.ac.kr>

### CODE REVIEW

File: lib/libutils/isoc/isalnum.c [hunk 1]
```diff
@@ -0,0 +1,10 @@
+// SPDX-License-Identifier: BSD-2-Clause
+/*
+ * Copyright (c) 2018, Linaro Limited
```
Comment (jforissier) [inline lib/libutils/isoc/isalnum.c]: s/2018/2019/

I am not sure you can attribute the copyright to Linaro. I think it should probably be your company/university or possibly yourself. @jbech-linaro any idea what's appropriate here?

Comment (jbech-linaro) [reply lib/libutils/isoc/isalnum.c]: > @jbech-linaro any idea what's appropriate here?

Agree, the contribution is not coming from a Linaro associated engineering, so we cannot have Linaro in the Copyright as it has been submitted here. So, we have two options:
1. @blukat29 add the company, organisation, university that he is working for to the existing patches.
or ...
2. With @blukat29 consent, keep the patch as it is and someone in Linaro (could be me or @jforissier for example), submits the patch with his `xyz@linaro.org` in the Signed-Off-By (we should also add Suggested-By in that case).

@blukat29 let us know that you prefer. For me I'm good with any of them.

Comment (blukat29) [reply lib/libutils/isoc/isalnum.c]: @jbech-linaro I would like to attribute my university. I will commit and push soon.

**Comment by jforissier** [comment]
`Acked-by: Jerome Forissier <jerome.forissier@linaro.org>`

**Pull request merged**